backend/src/files/types/export-file.type.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class ExportFileRequestBody {
+	@ApiProperty({ type: String, description: "export_type" })
+	exportType: "pdf" | "html" | "markdown";
+
+	@ApiProperty({ type: String, description: "markdown string" })
+	content: string;
+
+	@ApiProperty({ type: String, description: "File name" })
+	fileName: string;
+}
+
+export interface ExportFileResponseDto {
+	fileContent: Buffer;
+	mimeType: string;
+	fileName: string;
+}
```

Comment:
This name of this interface should not end with `Dto`. It should end with `XXXResponse`.

And please add swagger decorators.